windows/src/developer/TIKE/xml/kmw/packages.html

```diff
@@ -2,81 +2,51 @@
 <html>
 <head>
   <meta charset='utf-8'>
-  
-  <!-- Set the viewport width to match iOS device widths                         
+
+  <!-- Set the viewport width to match iOS device widths
   <meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=1.0,minimum-scale=1.0,user-scalable=no" />   -->
-  <meta name="viewport" content="width=device-width,user-scalable=no" /> 
+  <meta name="viewport" content="width=device-width,user-scalable=no" />
   <meta name="apple-mobile-web-app-capable" content="yes" />
-  
-  <!-- Enable IE9 Standards mode -->
-  <meta http-equiv="X-UA-Compatible" content="IE=edge" /> 
-    
-  <title>Keyman Developer Keyboard Package Installation Site</title>
-
-  <style type='text/css'>   
-    body {
-        padding-left: 10px;
-        margin-left: 12px;
-    }
 
-    h2 { 
-        font-family: sans-serif 
-    }
+  <!-- Enable IE9 Standards mode -->
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
 
-    .install-link { 
-      background: none repeat scroll 0 0 #CCCCCC;
-      border: 1px solid #444444;
-      border-radius: 4px;
-      box-shadow: 0 0 8px 0 rgba(0, 0, 0, 0.25);
-      color: #444444;
-      display: none;
-      font-family: Sans-serif;
-      font-size: 10pt;
-      margin: 0;
-      padding: 6px;
-      margin: 2px;
-      text-decoration: none;
-    }
-    
-    /** Mobile, Tablet **/
-    
-    #install-link {
-      font-weight: bold;
-    }
-    
-    .install-link .ios,
-    .install-link .android {
-      display: none;
-    }
+  <title>Keyman Developer Package Installation</title>
 
-    .touch-device #install-link,
-    .touch-device-ios #install-link-ios,
-    .touch-device-android #install-link-android {
-      display: inline-block;
-    }
+  <style type='text/css'>
+    @import url('test.css');
 
-    .touch-device-android .install-link .android,
-    .touch-device-ios .install-link .ios {
-      display: inline;
-    }
-  </style> 
+  </style>
 </head>
-  
+
 <body>
-  <h2>Available Packages for Installation</h2>
+  <div class='header'>
+    <div><h2>Install Packages</h2></div>
+    <div class='float-right'><a href='/' class='install-link'>Test keyboards online</a></div>
+    <div class='clear'></div>
+  </div>
+
+  <div id='content'>
+    <div id='top-toolbar'>
+      <div class='control' id='controls'>
+        <a href='https://keyman.com/go/developer/10.0/ios-app' target='_blank' class='install-link'>Install Keyman for iOS</a>
+        <a href='https://keyman.com/go/developer/10.0/android-app' target='_blank' class='install-link'>Install Keyman for Android</a>
```

Comment:
Do you want go/developer/12.0 links for these?

Reply:
Nah, it's not necessary, I think. The endpoints haven't changed.